siteconfig.json5

```diff
@@ -5,6 +5,9 @@
 
     // Configuration options for the QSS window
     qss: {
+        // Whether a user can open Settings pages and Control Panels of the operating system
+        osSettingsAvailable: true,
```

Comment:
Is there a JIRA for the core side of this implementation?

Reply:
In this case this option its not linked to core, just shows different side car information based on this variable.

Reply:
Where will the variable get its information from?

Reply:
You see the place, in the siteconfig file itself. Its a toggle do you want to show Windows Setting direct links in the side car or not. At least this is how I read the specs.